openfisca_dummy_country/model/model.py

```diff
@@ -35,10 +35,10 @@ class birth(Variable):
     definition_period = ETERNITY
```

Comment:
I agree this would be strange, but I don't think we ever explicitly imports model modules.